core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/configuration/security/jwt/JWTAuthenticationFilter.java

```diff
@@ -23,14 +25,26 @@ public JWTAuthenticationFilter(JWTAuthenticationConverter jwtAuthenticationConve
     }
 
     @Override
-    protected void doFilterInternal(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, FilterChain filterChain) throws ServletException, IOException {
-        Authentication authentication = jwtAuthenticationConverter.convert(httpServletRequest);
-        if (authentication == null) {
+    public void doFilterInternal(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, FilterChain filterChain) throws IOException, ServletException {
+        try {
+            Authentication authentication = jwtAuthenticationConverter.convert(httpServletRequest);
```

Comment:
only surround the jwtAuthenticationConverter.convert(httpServletRequest) with a try-catch clause

Reply:
bump